.github/workflows/linux.bash

```diff
@@ -1,12 +1,40 @@
 #!/bin/bash
 set -exo pipefail
 
-export PATH="/usr/lib/ccache:/usr/lib64/ccache:$PATH"
+export PATH="/usr/lib/ccache/bin:/usr/lib/ccache:/usr/lib64/ccache:$PATH"
 export CCACHE_DIR=/icinga2/ccache
 export CTEST_OUTPUT_ON_FAILURE=1
 CMAKE_OPTS=''
 
 case "$DISTRO" in
+  alpine:*)
+    # Install packages from Alpine package, just
+    # - LibreSSL instead of OpenSSL 3 and
+    # - no MariaDB or libpq as they depend on OpenSSL.
+    # https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/icinga2/APKBUILD
+    apk add \
+      bison \
+      boost-dev \
+      ccache \
+      cmake \
+      flex \
+      g++ \
+      libedit-dev \
+      libressl-dev \
+      ninja-build \
+      samurai \
```

Comment:
What's samurai for?

Reply:
As written in the comment above, this is derived from the [Alpine package build](https://github.com/Icinga/icinga-packaging/issues/93). Thus, I have no idea :)